plugins/modules/cmci_delete.py

```diff
@@ -12,11 +12,14 @@
 module: cmci_delete
 short_description: Delete CICS and CICSPlex SM resources
 description:
-  - Remove or discard definitional or installed CICS® and CICSPlex® SM resources from CICS
-    regions, by initiating DELETE requests via the CMCI REST API. The CMCI REST API can be configured
-    in CICSPlex SM or stand-alone regions (SMSS). For information about the API, see L(CMCI REST API,
+  - Remove or discard definitional or installed CICS® and CICSPlex® SM 
+    resources from CICS regions, by initiating DELETE requests via 
+    the CMCI REST API. The CMCI REST API can be configured in CICSPlex SM
+    or stand-alone regions (SMSS). For information about the API, see L(CMCI
```

Comment:
Do the links still work ok, if we split them across multiple lines?

Reply:
Yep~